### PR TITLE
fix: fix missing dependency

### PR DESCRIPTION
**Pull request by takayuki5168**
Signed-off-by: Takayuki Murooka <takayuki5168@gmail.com>

## Description

As written in the title

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by TakaHoribe — APPROVED** [review]
GTM

File: launch/tier4_perception_launch/package.xml [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
   <exec_depend>compare_map_segmentation</exec_depend>
   <exec_depend>detected_object_feature_remover</exec_depend>
+  <exec_depend>detected_object_feature_remover</exec_depend>
```
Comment (yukkysaito) [inline launch/tier4_perception_launch/package.xml:16]: mistake? already added.

Comment (takayuki5168) [reply launch/tier4_perception_launch/package.xml]: https://github.com/autowarefoundation/autoware.universe/pull/1891/commits/492c10d44317a8480f8f89804974383fd3a422c8
sorry

**Review by yukkysaito — APPROVED** [review]
LGTM

**Pull request merged**